wadas/ui/select_mode_dialog.py

```diff
@@ -55,4 +57,6 @@ def accept_and_close(self):
             OperationMode.cur_operation_mode_type = OperationMode.OperationModeTypes.AnimalDetectionAndClassificationMode
         elif self.ui.radioButton_bear_det_mode.isChecked():
             OperationMode.cur_operation_mode_type = OperationMode.OperationModeTypes.BearDetectionMode
+        elif self.ui.radioButton_custom_species_class_mode.isChecked():
+            OperationMode.cur_operation_mode_type = OperationMode.OperationModeTypes.CustomSpeciesClassificationMode
```

Comment:
Same here: match/case?

Reply:
not really, are different objects to check. not sure how to automate in match case

Reply:
Yes, it's a bit tricky. It requires a bit of redesign to achieve it, but it's not the right moment now.